app/auth/views/fido.py

```diff
@@ -153,6 +153,13 @@ def fido():
         webauthn_users, challenge
     )
     webauthn_assertion_options = webauthn_assertion_options.assertion_dict
+    try:
+        # HACK: We need to upgrade to webauthn > 1 so it can support specifying the transports
+        for credential in webauthn_assertion_options["allowCredentials"]:
+            del credential["transports"]
+    except KeyError:
+        # Should never happen but...
```

Comment:
Should we log a warning message here?

Reply:
In principle it should never happen as the library always adds it independently of whatever we pass it to it. If we fail to delete the only issue will be that we will limit the webauthn device to be the ones in the spec. So I think it's quite safe.